README.md

```diff
@@ -160,6 +160,12 @@ Option to define a custom file association on Windows.
 Caution: when you use `win.nsiTemplate` option, `win.fileAssociation` option should only work
 if the custom nsi template is based on the original one.
 
+### `win.encoding` *( optional )*
+Option to define a custom encoding for output template.nsi on Windows.
+Default value is 'UTF-8'.
+For available values see [ashtuchkin/iconv-lite's page](https://github.com/ashtuchkin/iconv-lite/wiki/Supported-Encodings).
+The packager.json config should always be UTF8 encoded.
```

Comment:
I changed that in README.md in the last commit, as I did not mean the packager.json but the used nsi template.

I think it is just as always with nodejs. As the BOM is not relevant for utf8 you can do either way If you supply a BOM, nodejs (At least this is what newer version of node do...) will just strip it in the decoded string. The same goes for writing an utf8 string back to disk with node: As the BOM is not relevant for utf8, node will not write it to the file.
(However I did not change anything with respect to this aspect in this pull request. Reading the template-file is just like before: A fs.readFile call of node's file module, reading an utf8 encoded file from disk. I just added the option to not dump it back as utf8 to the disk but encode it proir to this with a different encoding.)